server/bundler/loader.js

```diff
@@ -114,14 +114,10 @@ function splitTemplate( path, section ) {
 }
 
 function getPathRegex( pathString ) {
-	var regex;
-
 	if ( pathString === '/' ) {
 		return JSON.stringify( pathString );
 	}
-
-	regex = utils.pathToRegExp( pathString );
-
+	const regex = utils.pathToRegExp( pathString );
 	return '/' + regex.toString().slice( 1, -1 ) + '/';
```

Comment:
I'm not so concerned about knowing these details in a PR - I'm concerned about Joe or Mary Programmer knowing when they are tasked with jumping into this code and maintaining or modifying it so that they don't accidentally "fix" it and break everything.